packages/applications-service-api/__tests__/unit/controllers/documents.v3.test.js

```diff
@@ -183,4 +193,116 @@ describe('documentsV3 controller', () => {
 			});
 		});
 	});
+
+	describe('getDocumentByCaseReference', () => {
+		describe('NI', () => {
+			describe('dict test', () => {
```

Comment:
We can change the desc to something like "when document type is in wrong letter case"